.github/workflows/workflow.yml

```diff
@@ -2,6 +2,8 @@ name: Main workflow
 
 on:
   pull_request:
+    branches:
+      - master
```

Comment:
I don't think this will have any effect until it's merged into the master branch.

Reply:
I added this because the workflow didn't run. Then, after I added it, the workflow still didn't run.